spec/lib/pantry/helpers/request_verifier_spec.rb

```diff
@@ -0,0 +1,169 @@
+require 'rails_helper'
+
+RSpec.describe Pantry::Alexa::RequestVerifier do
+
+
+  describe ".verify" do
+
+    let(:body) do
+      " {\"session\":" +
+        "{\"sessionId\":\"SessionId.b4470d1c-67a0-4da2-98c7-853acead6773\",\"application\":{\"applicationId\":\"amzn1.ask.skill.057ceea6-1e3a-4edb-9a4f-f962f8cc0cac\"},\"attributes\":{},\"user\":{\"userId\":\"amzn1.ask.account.AEP3BKUW344CTHKJDR6P33Y2RJZL6AOSIIWPAEE23O6AIUHMLQ7VSG7NE2CHPDDDCMKF57M43ULCPZLG2U2CIN5NG7MWT6KCDVQS2PXDM5HQYN5J7A26JFPOT7E56W5NQ6S3GSDNINQQZEL5EHDFSXFBM3ST3M5CMK53IZHHQMBCYDA2EFD6OJL7HE6I7WGLSHSLMTN5I7YQDVQ\",\"accessToken\":null},\"new\":false}," +
+        "\"request\":" +
+        "{\"requestId\":\"EdwRequestId.9eaad9b4-48f7-4f6e-8ee9-42a4d7691d4e\",\"locale\":\"en-US\",\"type\":\"IntentRequest\",\"timestamp\":#{Time.now.to_i},\"intent\":{\"name\":\"Warning\",\"slots\":{}}},\"context\":{\"System\": {\"application\": {\"applicationId\": \"applicationId\"}, \"user\":{\"userId\": \"userId\",\"permissions\": {\"consentToken\": \"consentToken\"},\"accessToken\": \"accessToken\"},\"device\": { \"deviceId\": \"deviceId\",\"supportedInterfaces\":{} },\"apiEndpoint\":  \"apiEndpoint\"}}}"
+    end
+
+    let(:signature) do
+      "PGJpiD4Ex+3yha3u2e5NOOad6icEu4kv7oE/mTjHrxcXqsm58ESvBa5XtvRlBLHZXKb2oAFavTjKN1GAOz2vohw/Y+g2OIUdptLl6dTTQKFRrc8Wty/6+eKxYJ014nNnwYvT6YB9komGN4nxnTUiKSXLtxIpZynjyXAqP9cMV7su4OQcocT5fp5gs0ZQM+CHasLeGAUQdgU67HOdQ3XA+/U3Lrorr0x1e3YWidkogNxAA0dhpIDQbWuDNHSxoGBPUuCjOs/vOBrckH15yBoUd1CU3FTzDzCeF/soqnC3V1brCZhCbxh7kd3JUhj5YOLJyqcQ93pzO4ZPcJiX1R0fNw=="
+    end
+
+    let(:response) do
+      double('HTTP:Response', code: '200', body: 'Sample Body')
+    end
+
+    let(:http) do
+      double('Net::HTTP', :use_ssl= => true, :verify_mode= => true, start: true, request: response, finish: true)
```

Comment:
I'd stick to the modern hash syntax wherever you can (`use_ssl=: true` vs. `:use_ssl= => true`) unless you have non-symbol keys (`'string' => 'value'` or `MyClass => value`).